umap/static/umap/base.css

```
@@ -342,19 +342,24 @@ input.switch:checked ~ label:before {
 input.switch:checked ~ label:after {
     margin-left: 3em;
 }
-.button-bar {
+.button-bar, .umap-multiplechoice {
     margin-top: 5px;
     text-align: center;
     display: grid;
-    grid-gap: 7px;
     width: 100%
 }
+.button-bar {
+    grid-gap: 7px;
+}
 .button-bar.half {
     grid-template-columns: 1fr 1fr;
 }
-.button-bar.third {
+.button-bar.third, .umap-multiplechoice.by3 {
```

I couldn't find where `.third` is actually used - maybe it could be dropped?
```suggestion
.umap-multiplechoice.by3 {
```